git_machete/generated_docs.py

```diff
@@ -0,0 +1,938 @@
+
+        GitHub API server URL will be inferred from `git remote`.
+        You can override this by setting the following git config keys:
+           Remote name
+              E.g. `machete.github.remote` = `origin`
+           Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+           Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+        To do this, run `git config --local --edit` and add the following section:
+        <dim>
+          [machete "github"]
+              organization = <organization_name>
+              repository = <repo_name>
+              remote = <remote_name>
+        </dim>
+
+        <b>Options:</b>
+           <b>-c</b>, <b>--checkout-my-github-prs</b>
+              Checkout your open PRs into local branches.
+           <b>-y</b>, <b>--yes</b>
+              Don't ask for confirmation when deleting branches from git.
+
+        <b>Environment variables:</b>
+           `GITHUB_TOKEN`
+              GitHub API token.
+   """,
+    "config": """
+        Documentation about available `git machete` config keys and environment variables that change the command's default behavior.
+
+        Note: `config` is not a command as such, just a help topic (there is no `git machete config` command).
+
+        <b>Config keys:</b>
+           `machete.github.{remote,organization,repository}`:
+               When executing `git machete github <subcommand>` command, GitHub API server URL will be inferred from `git remote`.
+
+              Note
+
+              GitHub API server URL will be inferred from `git remote`.
+              You can override this by setting the following git config keys:
+
+                 Remote name
+              E.g. `machete.github.remote` = `origin`
+
+                 Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+
+                 Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+              To do this, run `git config --local --edit` and add the following section:
+
+              [machete "github"]
+                  organization = <organization_name>
+                  repository = <repo_name>
+                  remote = <remote_name>
+
+           `machete.overrideForkPoint.<branch>.{to,whileDescendantOf}`:
+              Executing `git machete fork-point --override-to=<revision> [<branch>]` sets up a fork point override for <branch>.
+              The override data is stored under `machete.overrideForkPoint.<branch>.to` and `machete.overrideForkPoint.<branch>.whileDescendantOf` git config keys.
+           `machete.status.extraSpaceBeforeBranchName`:
+
+              Note
+
+              To make it easier to select branch name from the `status` output on certain terminals
+              (e.g. Alacritty), you can add an extra
+              space between `└─` and `branch name` by setting `git config machete.status.extraSpaceBeforeBranchName true`.
```

Comment:
Whoops underlined elbow, pls sort all of them out

Reply:
(generally, all the comments to the specific format in generated_docs.py are obviously comments regarding the enitre algorithm of generating docs out of .rst)